cypress/integration/Dictionary/createDictionary.feature

```diff
@@ -0,0 +1,110 @@
+Feature: Dictionary Management
+
+  Scenario: A user to create a new dictionary
+    Given the user is logged in
+    And is on the user collections page
+    Then the user clicks on Create Dictionary Icon
+     Then the user should be able to create new dictionary
+
+  Scenario: A user should be able to create new dictionary by pressing on Create new dictionary icon
+    Given the user is on the Dictionaries page
+     When the user clicks on Create new dictionary icon
+     Then the create Dictionary form is filled
+     Then the user should be able to create new dictionary
```

Comment:
This doesn't seem like a useful scenario to have. The first part, i.e., that clicking the icon sends the user to the page is tested in the first scenario. The second part, i.e., that the dictionary can be created is tested in other scenarios.